Formula/zssh.rb

```diff
@@ -19,6 +19,9 @@ class Zssh < Formula
     sha256 cellar: :any_skip_relocation, yosemite:    "94280569f9e1c1deb9d8c3be4256cd501399fd51758f8e2ea6d77fd9f1b6ef2e"
   end
 
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "libtool"  => :build
```

Comment:
Since we're only running `autoreconf` on ARM, these deps should be scoped inside an `if Hardware::CPU.arm?` block as well.

Reply:
will do.

Reply:
done